### PR TITLE
Basic documentation for v1

### PR DESCRIPTION
**Pull request by thisandagain**

### CODE REVIEW

**Comment by thisandagain** [comment]
@k88hudson R?

File: README.md [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+Before you jump into the code you'll want to download, install, and configure the following:
 
-```
+- [Android Studio](http://developer.android.com/sdk)
+- [Node 0.12+](https://nodejs.org/) w/ [ES6 ("harmony") features enabled]()
+- [NPM 2.6+](https://www.npmjs.com/)
+
+#### Clone & Install Dependencies
+```bash
+git clone https://github.com/mozilla/webmaker-android
```
Comment (k88hudson) [inline README.md]: webmaker-app

File: README.md [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+cd webmaker-android
```
Comment (k88hudson) [inline README.md]: webmaker-app

File: README.md [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
 npm install
+```
+
+#### Android
+While the majority of `webmaker-android` is built using Web technologies, it runs within a native Android wrapper that is included as part of this codebase. If you would like to make changes to the wrapper or if you'd like to test the app, we recommend you use [Android Studio](http://developer.android.com/sdk/index.html).
```
Comment (k88hudson) [inline README.md]: webmaker-app

File: README.md [hunk 4]
```diff
@@ -1,13 +1,42 @@
...
+
+- Install and configure [Android Studio](http://developer.android.com/sdk)
+- Open Android Studio and select "Import Project"
+- Select the "webmaker-android" directory
```
Comment (k88hudson) [inline README.md]: webmaker-app

File: README.md [hunk 5]
```diff
@@ -1,13 +1,42 @@
...
+
+Once you have the project open, you can run it within an emulator or on any Android device with USB debugging enabled by selecting "Run 'app'" from the "Run" dropdown menu. For more information, please check out the [Android SDK documentation](http://developer.android.com/training/index.html).
+
+#### Web
+Each fragment within `webmaker-android` is actually just a web page! You can find all of the js, css, and static assets in the `./www_src/` directory. Static files in `./www_src/static/` will be copied to the main directory during build.
```
Comment (k88hudson) [inline README.md]: webmaker-app

**Comment by k88hudson** [comment]
Looks good, just needs a few name changes

**Comment by thisandagain** [comment]
I'm actually thinking this repo should change to `webmaker-android` as we'll likely have `webmaker-ffos` and `webmaker-ios` in the future. Thoughts?

**Comment by k88hudson** [comment]
Agreed although we should sort out the ops before we do that

**Comment by thisandagain** [comment]
:+1: I'll update the PR in the meantime.

**Pull request merged**